.github/workflows/run-tests.yml

```diff
@@ -43,11 +43,11 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y gdb-multiarch python3-dev python3-pip python3-wheel python3-setuptools git cmake gcc g++ pkg-config libglib2.0-dev
 
-    - name: Compile Capstone/Keystone/Unicorn and install Python modules
+    - name: Install Python modules (Capstone, Keystone, Unicorn, Ropper)
       if: steps.cache-libs.outputs.cache-hit != 'true'
```

Comment:
Changing the `cache-name` to `cache-deps` will result in never caching and always re-installing the pip packages

Reply:
Why? It'll just cause the cache name to change one, to be more accurate, and then in the future we'll only use this new name.

Reply:
Nevermind, I see what you mean, I forgot to change it here. Nice find :)